### ISSUE TITLE
Mac Ventura 13.0.1 can not start kube-explorer

### ISSUE THREAD

**Issue by cdmikechen** [niusmallnan/kube-explorer#30]
When I upgraded to the latest mac os (Ventura 13.0.1), I found that the kube-explorer would not start. 
I have upgraded to the latest version with the same problem.

```shell
./kube-explorer --kubeconfig=/Users/xxx/.kube/config --http-listen-port=9898 --https-listen-port=0
[1]    20848 segmentation fault  ./kube-explorer --kubeconfig=/Users/xxxd/.kube/config
```

logs in `~/Library/Logs/DiagnosticReports/`
```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Incident Identifier: 50D15C8C-FBA9-40CC-BB21-BE0F2D726E48
CrashReporter Key:   F8B6804D-137B-BC28-CFC0-E3EE7FE4EA53
Hardware Model:      Macmini8,1
Process:             kube-explorer [21387]
Path:                /Users/USER/*/kube-explorer
Identifier:          kube-explorer
Version:             ???
Code Type:           X86-64 (Native)
Role:                Unspecified
Parent Process:      Exited process [21385]
Coalition:           com.googlecode.iterm2 [1424]
Responsible Process: iTerm2 [3441]

Date/Time:           2022-11-27 12:19:37.9474 +0800
Launch Time:         2022-11-27 12:19:37.6098 +0800
OS Version:          macOS 13.0.1 (22A400)
Release Type:        User
Report Version:      104

Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Subtype: KERN_INVALID_ADDRESS at 0x00000000000000c8
Exception Codes: 0x0000000000000001, 0x00000000000000c8
VM Region Info: 0xc8 is not in any region.  Bytes before following region: 140737488269112
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      UNUSED SPACE AT START
--->  
      shared memory            7ffffffeb000-7ffffffec000 [    4K] r-x/r-x SM=SHM  
Termination Reason: SIGNAL 11 Segmentation fault: 11
Terminating Process: exc handler [21387]

Highlighted by Thread:  0

Backtrace not available

No thread state (register information) available

Binary Images:
               0x0 - 0xffffffffffffffff ??? (*) <00000000-0000-0000-0000-000000000000> ???

Error Formulating Crash Report:
_dyld_process_info_create failed with 30
dyld_process_snapshot_get_shared_cache failed
Failed to create CSSymbolicatorRef - corpse still valid ¯\_(ツ)_/¯
thread_get_state(PAGEIN) returned 0x10000003: (ipc/send) invalid destination port
thread_get_state(EXCEPTION) returned 0x10000003: (ipc/send) invalid destination port
thread_get_state(FLAVOR) returned 0x10000003: (ipc/send) invalid destination port

EOF

-----------
Full Report
-----------

{"app_name":"kube-explorer","timestamp":"2022-11-27 12:19:44.00 +0800","app_version":"","slice_uuid":"00000000-0000-0000-0000-000000000000","build_version":"","platform":0,"share_with_app_devs":0,"is_first_party":1,"bug_type":"309","os_version":"macOS 13.0.1 (22A400)","roots_installed":0,"incident_id":"50D15C8C-FBA9-40CC-BB21-BE0F2D726E48","name":"kube-explorer"}
{
  "uptime" : 51000,
  "procRole" : "Unspecified",
  "version" : 2,
  "userID" : 0,
  "deployVersion" : 210,
  "modelCode" : "Macmini8,1",
  "coalitionID" : 1424,
  "osVersion" : {
    "train" : "macOS 13.0.1",
    "build" : "22A400",
    "releaseType" : "User"
  },
  "captureTime" : "2022-11-27 12:19:37.9474 +0800",
  "incident" : "50D15C8C-FBA9-40CC-BB21-BE0F2D726E48",
  "pid" : 21387,
  "cpuType" : "X86-64",
  "roots_installed" : 0,
  "bug_type" : "309",
  "procLaunch" : "2022-11-27 12:19:37.6098 +0800",
  "procStartAbsTime" : 51436034057640,
  "procExitAbsTime" : 51436371440052,
  "procName" : "kube-explorer",
  "procPath" : "\/Users\/USER\/*\/kube-explorer",
  "parentProc" : "Exited process",
  "parentPid" : 21385,
  "coalitionName" : "com.googlecode.iterm2",
  "crashReporterKey" : "F8B6804D-137B-BC28-CFC0-E3EE7FE4EA53",
  "responsiblePid" : 3441,
  "responsibleProc" : "iTerm2",
  "bridgeVersion" : {"build":"20P420","train":"7.0"},
  "sip" : "enabled",
  "vmRegionInfo" : "0xc8 is not in any region.  Bytes before following region: 140737488269112\n      REGION TYPE                    START - END         [ VSIZE] PRT\/MAX SHRMOD  REGION DETAIL\n      UNUSED SPACE AT START\n--->  \n      shared memory            7ffffffeb000-7ffffffec000 [    4K] r-x\/r-x SM=SHM  ",
  "exception" : {"codes":"0x0000000000000001, 0x00000000000000c8","rawCodes":[1,200],"type":"EXC_BAD_ACCESS","signal":"SIGSEGV","subtype":"KERN_INVALID_ADDRESS at 0x00000000000000c8"},
  "termination" : {"flags":0,"code":11,"namespace":"SIGNAL","indicator":"Segmentation fault: 11","byProc":"exc handler","byPid":21387},
  "vmregioninfo" : "0xc8 is not in any region.  Bytes before following region: 140737488269112\n      REGION TYPE                    START - END         [ VSIZE] PRT\/MAX SHRMOD  REGION DETAIL\n      UNUSED SPACE AT START\n--->  \n      shared memory            7ffffffeb000-7ffffffec000 [    4K] r-x\/r-x SM=SHM  ",
  "extMods" : {"caller":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"system":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"targeted":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"warnings":0},
  "usedImages" : [
  {
    "size" : 0,
    "source" : "A",
    "base" : 0,
    "uuid" : "00000000-0000-0000-0000-000000000000"
  }
],
  "legacyInfo" : {
  "threadHighlighted" : 0
},
  "trialInfo" : {
  "rollouts" : [

  ],
  "experiments" : [

  ]
},
  "reportNotes" : [
  "_dyld_process_info_create failed with 30",
  "dyld_process_snapshot_get_shared_cache failed",
  "Failed to create CSSymbolicatorRef - corpse still valid ¯\\_(ツ)_\/¯",
  "thread_get_state(PAGEIN) returned 0x10000003: (ipc\/send) invalid destination port",
  "thread_get_state(EXCEPTION) returned 0x10000003: (ipc\/send) invalid destination port",
  "thread_get_state(FLAVOR) returned 0x10000003: (ipc\/send) invalid destination port"
]
}


```

**Comment by orangedeng** [comment]
Please refer to this issue: https://github.com/cnrancher/autok3s/issues/516
It is the UPX [issue](https://github.com/upx/upx/issues/612).

**Comment by niusmallnan** [comment]
Fixed by v0.3.2
